app/views/cities/forbes_show.html.erb

```diff
@@ -25,7 +25,8 @@
   	<% elsif @city.cold_water? %>
   	  <%= render partial: 'cities/show/cold_water' %>
   	<% else %>
-  	  <%= render partial: 'cities/show/active_waitlisted' %>
+  	  <!-- <%= render partial: 'cities/show/active_waitlisted' %> -->
+      <%= render partial: 'cities/show/active' %>
```

Comment:
This *requires* that new users stop being waitlisted, as the site acts funny if a user who's on the waitlist can see tea times and signs up for them. They end of getting signed up, but the site tells them they can't do it (even though they...actually get signed up)

However, if a new user goes to a pending city page or creates a new city, the flow on the new site should not change. They should still see `cold_water` and `unconfirmed` for cities that fall under those categories.